ftp.go

```diff
@@ -33,11 +34,14 @@ type ServerConn struct {
 	// Timezone that the server is in
 	Location *time.Location
 
+	netConn       net.Conn
```

Comment:
This variable is needed for the "upgrade" functionality to work (because we need the bare network connection for that).

It _could_ be removed if the `StartTLS`  functionality is replaced by just having `DialTLS` (see other comment).